task/source-build/0.1/source-build.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: source-build
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
```

Comment:
Does `hacbs` has special meaning? If no, please remove.

Reply:
Just wrote it as the same tags within other tasks. Removed it from source build task.